fuzzing/private/oss_fuzz/package.bzl

```diff
@@ -20,7 +20,24 @@ def _oss_fuzz_package_impl(ctx):
     output_archive = ctx.actions.declare_file(ctx.label.name + ".tar")
     binary_info = ctx.attr.binary[FuzzingBinaryInfo]
 
-    action_inputs = [binary_info.binary_file]
+    action_inputs = binary_info.binary_runfiles.files.to_list()
```

Comment:
Why is this variable called "action_inputs"?

Reply:
It was called that before I touched the code and is appended to below, but I am (ab)using its intermediate value to deal with `binary_runfiles` here. I will introduce a new variable that holds the intermediate result only and is not mutated.